packages/buckram/assets/styles/variables/_specials.scss

```diff
@@ -964,16 +964,77 @@ $examples-background: #f3e1e3 !default;
 /// @type String
 $examples-header-background: #7a333a !default;
 
+// Contributors
+
+/// Margin top for elements with a class of `contributors`.
+/// @type String
+$contributors-margin-top: 2rem !default;
+
+/// Margin bottom for elements with a class of `contributors`.
+/// @type String
+$contributors-margin-bottom: 2rem !default;
+
+/// Text indent for elements with a class of `contributors`.
+/// @type String
+$contributors-text-indent: 0 !default;
+
+/// Text align for elements with a class of `contributors`.
+/// @type String
+$contributors-text-align: left !default;
+
+/// Float property for contributor profile pictures.
+/// @type String
+$contributor-profile-picture-float: left !default;
+
+/// Margin-top for contributor profile pictures.
+/// @type String | Map
+$contributor-profile-picture-margin-top: (
```

Comment:
@greatislander These mapped margin values for various stylesheets aren't being parsed correctly. I tried to copy what I saw elsewhere in Buckram, but I don't think margin values work in the same way and I don't know why. Here's what I see in my browser console
![Screenshot from 2021-09-17 09-24-26](https://user-images.githubusercontent.com/13485451/133821888-85ce1d9d-48f3-47a3-be06-489b87692f50.png)
:

Reply:
Solved! Needed to use if-map-get.